configuration/pih/htmlforms/consult.xml

```diff
@@ -207,19 +207,12 @@
             width: 90%;
         }
 
-        .medication-instructions input { 
+        .medication-instructions input {
             width: 85%;
         }
 
     </style>
 
-    <script>
-        // I don't really understand why this assignment `$j = jq` is required, but for some
-        // reason a bunch of Javascript gets included in the page that expects `$j` to exist,
-        // but doesn't define `$j` itself. *shrug emoji*
-        var $j = jQuery;
-    </script>
```

Comment:
I think I've fixed this so this is no longer needed... the Medication section was likely failing before if this wasn't present... I removed this and tested the Medication section locally and it worked... @brandones  do you happen to remember what else was failing so we can test?

Reply:
I don't, no.